js/app.jsx

```diff
@@ -486,10 +486,24 @@ var Job = React.createClass({
       ['Output', cleanJobPath(job.conf.output)],
     ];
     var bytes = {
-      read: bytesFormat(job.counters.get('hdfs.bytes_read').map),
-      written: bytesFormat(job.counters.get('hdfs.bytes_written').total),
+      hdfs_read: bytesFormat(job.counters.get('hdfs.bytes_read').map),
+      s3_read: bytesFormat(job.counters.get('s3.bytes_read').map),
+      file_read: bytesFormat(job.counters.get('file.bytes_read').map),
+      total_read: bytesFormat(
+        job.counters.get('hdfs.bytes_read').map +
+        job.counters.get('s3.bytes_read').map +
+        job.counters.get('file.bytes_read').map),
+      hdfs_written: bytesFormat(job.counters.get('hdfs.bytes_written').total),
+      s3_written: bytesFormat(job.counters.get('s3.bytes_written').total),
+      file_written: bytesFormat(job.counters.get('file.bytes_written').total),
+      total_written: bytesFormat(
+        job.counters.get('hdfs.bytes_written').total +
+        job.counters.get('s3.bytes_written').total +
+        job.counters.get('file.bytes_written').total),
```

Comment:
It might be nice to iterate over the values and call `bytesFormat` on all of them rather than needing to remember the function call on every value.